torch/onnx/symbolic_opset9.py

```diff
@@ -2217,20 +2217,18 @@ def gather(g, self, dim, index, sparse_grad=False):
 
 @parse_args('v', 'is', 'b', 'i')
 def _var_mean(g, input, dim, unbiased, keepdim):
-    sqrd = g.op("Mul", input, input)
     if dim is None:
-        sqrdmean = g.op("ReduceMean", sqrd, keepdims_i=0)
         mean = g.op("ReduceMean", input, keepdims_i=0)
         num_elements = numel(g, input)
     else:
-        sqrdmean = g.op("ReduceMean", sqrd, axes_i=dim, keepdims_i=keepdim)
         mean = g.op("ReduceMean", input, axes_i=dim, keepdims_i=keepdim)
         redudced_dims = g.op("Shape", input)
         # dim could contain one or multiple dimensions
         redudced_dims = g.op("Gather", redudced_dims, g.op("Constant", value_t=torch.tensor(dim)), axis_i=0)
         num_elements = g.op("ReduceProd", redudced_dims, keepdims_i=0)
-    meansqrd = g.op("Mul", mean, mean)
-    var = g.op("Sub", sqrdmean, meansqrd)
+    sub_v = g.op("Sub", input, mean)
+    sqr_sub = g.op("Mul", sub_v, sub_v)
+    var = g.op("ReduceMean", sqr_sub, axes_i=dim, keepdims_i=0)
```

Comment:
error may from here `keepdims_i`

Reply:
Thanks, yes, should be `keepdims_i=keepdims`, running tests now.